package.json

```diff
@@ -20,7 +20,7 @@
     "watch": "pnpm --filter=@pnpm/fetch run compile && pnpm --filter=pnpm run compile --watch",
     "make-lcov": "shx mkdir -p coverage && lcov-result-merger './packages/*/coverage/lcov.info' 'coverage/lcov.info'",
     "update-manifests": "pnpm run meta-updater && pnpm install",
-    "meta-updater": "pnpm --filter=@pnpm-private/updater compile && pnpm --config.use-node-version=16.4.0 exec meta-updater",
+    "meta-updater": "pnpm --filter=@pnpm-private/updater compile && pnpm exec meta-updater",
```

Comment:
why do you want to run it with node.js 16 if meta-updater v2 has dropped older node.js versions support?

Reply:
I don't, I want to remove the requirement of running it with node.js 16, sorry if my message was unclear.

Reply:
Sorry, I misread the diff.